[2021-12-22 17:42:06,569] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: data_pipeline1.sketch scheduled__2021-12-20T00:28:00+00:00 [queued]>
[2021-12-22 17:42:06,573] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: data_pipeline1.sketch scheduled__2021-12-20T00:28:00+00:00 [queued]>
[2021-12-22 17:42:06,573] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-22 17:42:06,573] {taskinstance.py:1242} INFO - Starting attempt 1 of 4
[2021-12-22 17:42:06,573] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-22 17:42:06,582] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): sketch> on 2021-12-20 00:28:00+00:00
[2021-12-22 17:42:06,585] {standard_task_runner.py:52} INFO - Started process 31377 to run task
[2021-12-22 17:42:06,589] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'data_pipeline1', 'sketch', 'scheduled__2021-12-20T00:28:00+00:00', '--job-id', '169', '--raw', '--subdir', '/home/myamya/project/dockers/airflow/dags/hello_world.py', '--cfg-path', '/tmp/tmpmq5gwxmr', '--error-file', '/tmp/tmpjrtj99gh']
[2021-12-22 17:42:06,589] {standard_task_runner.py:77} INFO - Job 169: Subtask sketch
[2021-12-22 17:42:06,616] {logging_mixin.py:109} INFO - Running <TaskInstance: data_pipeline1.sketch scheduled__2021-12-20T00:28:00+00:00 [running]> on host myamya
[2021-12-22 17:42:06,645] {taskinstance.py:1412} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=yangjae
AIRFLOW_CTX_DAG_ID=data_pipeline1
AIRFLOW_CTX_TASK_ID=sketch
AIRFLOW_CTX_EXECUTION_DATE=2021-12-20T00:28:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-20T00:28:00+00:00
[2021-12-22 17:42:06,645] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2021-12-22 17:42:06,645] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'docker run -it -v /home/myamya/project/image_files/:/home/myamya/project/image_files/ sketch:0.4 /bin/bash /home/myamya/project/init.sh ']
[2021-12-22 17:42:06,649] {subprocess.py:85} INFO - Output:
[2021-12-22 17:42:26,529] {subprocess.py:93} INFO - Command exited with return code 0
[2021-12-22 17:42:26,563] {taskinstance.py:1270} INFO - Marking task as SUCCESS. dag_id=data_pipeline1, task_id=sketch, execution_date=20211220T002800, start_date=20211222T084206, end_date=20211222T084226
[2021-12-22 17:42:26,610] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-22 17:42:26,629] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
